wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/WireRun.kt

```diff
@@ -327,12 +327,14 @@ class WireRun(
     }
 
     for (emittingRules in targetToEmittingRules.values) {
-      if (emittingRules.unusedIncludes().isNotEmpty()) {
-        logger.unusedIncludesInTarget(emittingRules.unusedIncludes())
+      val unusedIncludes = emittingRules.unusedIncludes()
+      if (unusedIncludes.isNotEmpty() && unusedIncludes != setOf("*")) {
```

Comment:
Maybe worth some context here, similar to the comment on the test, to explain how it could be `["*"]` and why to ignore it?

Reply:
Yeah, great idea